changelog/@unreleased/pr-1649.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: add artifacts manifest extension
```

Comment:
nice - could we include a mention (and maybe example) in the README so that if people search for this they'll find something?